docs/source/noise/index.rst

```diff
@@ -38,45 +39,72 @@ Currently, errors added in different columns are independent of each other.
 Available Noise Types
 ---------------------
 
-Pseudopeople applies the different types of row-based and column-based noise in
-the following order to mimic the data generation process by which a real dataset
-might be corrupted. The table lists all the available noise types, but not every
+The table lists all the available noise types, but not every
 type of noise will necessarily be applied to every dataset or every column. The
 configuration determines which noise types are actually used. The "Noise Type"
 column shows the name of the noise type in the configuration system.
 
-.. list-table:: Types of noise in order of application
-  :widths: 1 2 5 1
+.. list-table:: Types of row-based noise (``row_noise``)
+  :widths: 1 2 5 
   :header-rows: 1
 
   * - Noise Type
-    - Description
-    - Example
-    - Order
-  * - ``row_noise``
-    - **Row-based noise**
-    - ---
-    - ---
-  * - ``omit_row``
-    - Omit a row
+    - Config key 
+    - Example cause
+  * - Omit a row
+    - ``omit_row``
     - Neglecting to file a tax form on time
-    - 1
-  * - ``column_noise``
-    - **Column-based noise**
-    - ---
-    - ---
-  * - ``leave_blank``
-    - Leave a field blank
+  * - Duplicate a row
+    - ``duplicate_row``
+    - College student being reported both at their university and their guardian's
+      home address
+
+
+.. list-table:: Types of column-based noise (``column_noise``)
+  :widths: 1 2 5 
+  :header-rows: 1
+
+  * - Noise Type
+    - Config key
+    - Example cause
+  * - Leave a field blank
+    - ``leave_blank``
     - Forgetting to write your name on the designated line
-    - 2
-  * - ``choose_wrong_option``
-    - Choose the wrong option from a fixed set of options
+  * - Choose the wrong option from a fixed set of options
+    - ``choose_wrong_option``
     - Marking the "Male" box when you meant "Female"
-    - 3
-
-.. todo::
+  * - Optical character recognition (OCR) error
```

Comment:
I'm wondering what we should do about these, which we won't have for CODS? @aflaxman any thoughts, should we add an asterisk or something? Remove them altogether for now?

Reply:
Oh I didn't realize these hadn't been implemented yet. If we want to follow same thing we did for the datasets page, I could add a 'coming soon' statement to the noises that haven't been added yet (i.e., like what I said for the 1040 tax forms).

Reply:
As discussed in the documentation session, I think we should remove the unimplemented things for now (but save your work somewhere if you've already written something!)

I don't think OCR noise is implemented yet.